perllib/FixMyStreet/Cobrand/Bromley.pm

```diff
@@ -364,5 +369,133 @@ sub munge_load_and_group_problems {
     }
 }
 
-1;
+sub munge_around_category_where {
+    my ($self, $where) = @_;
+    $where->{extra} = [ undef, { -not_like => '%Waste%' } ];
+}
+
+sub munge_reports_category_list {
+    my ($self, $categories) = @_;
+    @$categories = grep { grep { $_ ne 'Waste' } @{$_->groups} } @$categories;
+}
+
+sub munge_report_new_contacts {
+    my ($self, $categories) = @_;
 
+    return if $self->{c}->action =~ /^hercules/;
+
+    @$categories = grep { grep { $_ ne 'Waste' } @{$_->groups} } @$categories;
+    $self->SUPER::munge_report_new_contacts($categories);
+}
+
+sub bin_addresses_for_postcode {
+    my $self = shift;
+    my $pc = shift;
+
+    my $echo = $self->feature('echo');
+    $echo = Integrations::Echo->new(%$echo);
+    my $result = $echo->FindPoints($pc);
+
+    my $points = $result->{PointInfo};
```

Comment:
Need some error checking here to handle when you get no results, e.g a non bromley postcode.

Reply:
e.g for me BR1 3UZ gets a 500 error page but BR1 3UM gets a did not recognise postcode